client/web/src/components/popovers/ColorPicker.vue

```diff
@@ -14,13 +14,14 @@
 
 <style lang="scss">
 .popover-color-picker {
-	--hue: #ff0000;
+	--hue-picker-color: #ff0000;
```

Comment:
Shouldn't this say `--saturation-picker-color`?

Reply:
Looks good! As a small nitpick, would you mind changing `--saturation-picker-color` to `--saturation-picker-hue` which is technically more accurate? You probably didn't see my edit to the above description when I added the second sentence. I'll go ahead and hit approve now and you're ready to merge!